.travis.yml

```diff
@@ -48,6 +57,11 @@ jobs:
       php: 7.2
       script: bin/roave-backward-compatibility-check --from=master --to=HEAD
 
+    - stage: Build release PHAR
```

Comment:
Is it possible to add a stage that builds the phar anyway? Wouldn't want to have this process randomly break

Reply:
Possibly, yes.
We can just leave the `deploy` with `on: tags: true` and leave this stage run always. :+1:

Reply:
As long as it doesn't push the artifact :-P

Reply:
Updated.

Reply:
> As long as it doesn't push the artifact :-P

It shouldn't, but verify yourself:
https://docs.travis-ci.com/user/deployment/releases/#on.tags-condition
https://docs.travis-ci.com/user/deployment/#Conditional-Releases-with-on%3A

> When all conditions specified in the on: section are met, your build will deploy.
> tags can be true, false or any other string:
>     tags: true: deployment is triggered if and only if $TRAVIS_TAG is set.